src/Carbon/Lang/pt_BR.php

```diff
@@ -28,4 +28,13 @@
     'from_now' => 'em :time',
     'after' => 'após :time',
     'before' => ':time atrás',
+    'diff_now' => 'hoje',
```

Comment:
`diff_now` only apply for a date less than 1 second ago. But there is no "today" word. In english for example if the date is 2 hours old, you get "2 hours ago". So are you sure there is no better fit for "just now"?

Reply:
Thanks for your reply, in this case the better word is "agora"